### ISSUE TITLE
Umineko not fully displaying 960 x 540 resolution

### ISSUE THREAD

**Issue by Tiresias33** [MyLegGuy/Higurashi-Vita#41]
As the title says, the PS Vita renders the backgrounds with two vertical black lines on each side of the background image. Manually setting each background resolution to 960 x 540 nor any other resolution (bigger or smaller) always yields this result. It doesn't help that the Umineko Converter doesn't support converting to PS Vita resolutions.
I noticed that other games that had 640 x 480 background resolutions were able to display them in the Vita at full screen.
Any hints as to the issue?

**Comment by MyLegGuy** [comment]
Fullscreen VNDS games? I've never heard of such a thing.
> Manually setting each background resolution to 960 x 540 nor any other resolution
Changing the background resolution? How do you do that? Is it an option in the Umineko VNDS converter? If so, then maybe the converter insists on keeping the 4:3 aspect ratio regardless of your input. Try checking the output images to see what size they actually are.

**Comment by Tiresias33** [comment]
Here's a "fullscreen" version of Higurashi, based on your tutorial I followed here: https://wololo.net/talk/viewtopic.php?f=116&t=48223
![higurashi 960 x 540](https://user-images.githubusercontent.com/82339301/114459104-f0105780-9b94-11eb-9825-15242e5ba7fa.png)
The resolution here is 960 x 540.
I found other games to run at this resolution as well, such as Saya No Uta and Ever 17.

In the Umineko converter, all the resolutions that you're able to convert the game to have a 4:3 aspect ratio (from 256x192 up to 1280 x 800), none being 16:9.
![umineko converter](https://user-images.githubusercontent.com/82339301/114500370-bebb7a00-9bdc-11eb-958d-09ca78d3058f.png)

The output background files are usually 640 x 480, and while I tried batch converting them all to 960 x 540 and modifying the "img.ini" file to reflect the changes, it still comes out at 4:3 aspect ratio.
I'll try to see if I can find images with a 16:9 aspect ratio to replace them.
While messing around with two copies of Ever 17, (displayed at 4:3 ratio vs another VNDS converted with 16:9 ratio), I just realized that replacing all the images with a 16:9 ratio makes the game run at that ratio. Interestingly enough, if just one image is at a different resolution, it doesn't mess with the entire game's display (which I thought was the case).
Perhaps I didn't replace all the images, or perhaps the game itself is being too stubborn to change to 16:9. I'll keep messing around to see if I can find something out of this. I would really like to play the novel in 960 x 540 resolution, if possible.

**Comment by Tiresias33** [comment]
Update: I made some progress and got the backgrounds to show up properly:
![umineko at 960 x 540](https://user-images.githubusercontent.com/82339301/114507700-596d8600-9be8-11eb-889f-59482cb26c36.png)
-Yeah, the characters don't show up properly, but that's because I haven't edited them to the proper size.
I think I forgot to recompress my images back into the "background.zip" files prior to converting to VNDS again (I'm using your VNDSConverter).  Yeah, it's a silly mistake, but things can get really confusing when a converter can't do all the work for you lol.


However, I will say that when I did attempt to resize some of the image files, they did force themselves back to another resolution. For example, in attempting to manually convert an image file to 960 x 540, they'll be converted to 720 x 540 (which is 4:3). I guess some images are just set to 4:3 by default, outside of the Umineko converter (I think the source of the images are what dictates that).
Anyways, I think I can take it from here. Your Higurashi Windows tool and your VNDS Converter work like a charm, by the way. Unless you want to add some comments or questions, I think we can have this closed.
